openwisp_controller/config/migrations/0035_custom_sorted_m2m_field.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.8 on 2021-04-23 22:17
+
+from django.conf import settings
+from django.db import migrations
+
+import openwisp_controller.config.sortedm2m.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('config', '0034_template_required'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='config',
+            name='templates',
+            field=openwisp_controller.config.sortedm2m.fields.SortedManyToManyField(
+                blank=True,
+                help_text='configuration templates, applied from first to last',
+                related_name='config_relations',
+                to=settings.CONFIG_TEMPLATE_MODEL,
+                verbose_name='templates',
+            ),
+        ),
+    ]
```

Comment:
please do not create a new migration but modify the existing migrations, we do this in order to limit the amount of migration files, which would otherwise skyrocket.